x-pack/legacy/plugins/apm/public/components/shared/KueryBar/index.tsx

```diff
@@ -48,11 +54,23 @@ export function KueryBar() {
   });
   const { urlParams } = useUrlParams();
   const location = useLocation();
+  const matchedRoutes = useMatchedRoutes();
+
   const apmIndexPatternTitle = chrome.getInjected('apmIndexPatternTitle');
   const indexPatternMissing =
     !state.isLoadingIndexPattern && !state.indexPattern;
   let currentRequestCheck;
 
+  const exampleMap: { [key: string]: string } = {
+    [TRANSACTIONS]: 'transaction.duration.us > 300000',
+    [ERRORS]: 'http.response.status_code >= 400',
+    [METRICS]: 'process.pid = "1234"'
+  };
+
+  const queryExample =
+    matchedRoutes.map(({ name }) => exampleMap[name]).find(Boolean) ||
+    'transaction.duration.us > 300000 AND http.response.status_code >= 400';
```

Comment:
i know i made the first suggestion for setting `queryExample` this way, but maybe we can add a line comment to demystify this compact definition.

Reply:
`// sets queryExample to the first matched example query, else default example`

Reply:
done!